Sources/URLSessionDecodable/URLSessionDecodable.swift

```diff
@@ -28,29 +28,39 @@ extension URLSession {
         completionHandler: @escaping (Result<T, URLSessionDecodableError>) -> Void
     ) -> URLSessionDataTask {
         let request = self.request(with: url, method: method, parameters: parameters, headers: headers)
+        return decodable(request: request, decoder: decoder, completionHandler: completionHandler)
+    }
+    // swiftlint:enable function_parameter_count
+
+    // swiftlint:disable force_unwrapping
```

Comment:
It would be better to avoid it

Reply:
I tried to avoid it, but then I realized that there's no way to create `URLRequest` without `URL` (at least now). Thus I decided to ignore force-unwrapping.

I'll remove force-unwrapping and will use `URLSessionDecodableError.unknown` to handle case when there's no url. I don't want to create a special error for this case, because it seems very unlikely.